### ISSUE TITLE
Instahack not working

### ISSUE THREAD

**Issue by khanfaisal313313** [avramit/Instahack#59]
Showing

Module not found error: No module named 'socks'

**Comment by newhavbsb** [comment]
Install socks

**Comment by khanfaisal313313** [comment]
How to install socks

**Comment by newhavbsb** [comment]
pip install pysocks

**Comment by newhavbsb** [comment]
I am getting Module not found error: No module named 'proxybrokrer'

**Comment by avramit** [comment]
please install all the moduels required from the readme file

**Comment by Sanif007** [comment]
Sir I installed all but at time of installing proxybroker it is showing something wierd

**Comment by Udoy2** [comment]
Yes every module install successfully but erorr came in proxy broker we need help

**Comment by mahetaparth1** [comment]
Sir i did e erything and lots of tries. Bt proxybroker error irritating me. So plz help me to get out the somution
